product_supplierinfo_for_customer_elaboration/models/sale_order_line.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Tecnativa - Carlos Roca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.onchange("product_id")
+    def product_id_change(self):
+        res = super().product_id_change()
+        customerinfo = self.product_id.customer_ids.filtered(
+            lambda pc: pc.elaboration_id and pc.name == self.order_id.partner_id
+        )[:1]
+        if not customerinfo:
```

Comment:
This doesn't make sense, as if no `customerinfo`, filter anything will be again no `customerinfo`

Reply:
Can you please check this?